Makefile

```diff
@@ -1,3 +1,7 @@
+KEYS_MONITOR_IMAGE_NAME=keysmonitor
+KEYS_MONITOR_IMAGE_NAME_TAG=latest
```

Comment:
At some point, perhaps a specific tag will also be desired. For now, all good to push to "latest" tag of the Docker "repository".

Reply:
I saw that the Docker image builder also pushes the proper version and the 'latest' version. 
![image](https://github.com/multiversx/mx-chain-keys-monitor-go/assets/36724586/9e9b8dc2-b496-46eb-a468-f54c2ab3b2c8)
(latest and the v1.0.4-alpha4 have the same hash)

Reply:
Everything seems to be in-place (since the publish is performed by the GH workflow) :+1: